webserver/__init__.py

```diff
@@ -27,15 +20,18 @@ def create_app():
     from webserver.loggers import init_loggers
     init_loggers(app)
 
-    # Kafka connection
-    from kafka_connection import init_kafka_connection
-    init_kafka_connection(app.config['KAFKA_CONNECT'])
-
     # Database connection
     from db import init_db_connection
     init_db_connection(app)
     messybrainz.db.init_db_engine(app.config['MESSYBRAINZ_SQLALCHEMY_DATABASE_URI'])
 
+    # Connections to external servers
+    @app.before_request
+    def before_reqeust():
+        g.kafka = kafka_connection.init_kafka_connection(app.config['KAFKA_CONNECT'])
+        g.listenstore = listenstore_connection.init_listenstore(current_app.config['CASSANDRA_SERVER'], current_app.config['CASSANDRA_KEYSPACE'])
+
```

Comment:
Does this really make sense? So far only one of our endpoints talks to kafka, but this will require every connection to have a kafka handle. That's kinda kafkaesque, no? 

Maybe we should make this a decorator that we apply to kafka using endpoints.